docs/SmartStore.md

```diff
@@ -242,3 +242,16 @@ See [indexes.conf](https://docs.splunk.com/Documentation/Splunk/latest/Admin/Ind
 | maxCacheSize | max_cache_size  | [cachemanager] |
 | evictionPolicy |eviction_policy  |[cachemanager] |
 | evictionPadding | eviction_padding  |[cachemanager] |
+
+## Additional configuration
+
+If there is a need to configure Smartstore or Index related configs in addition to the above, this can be achieved by including the additional configs via Apps:
+1. Create an App with the additional configuration
+For example, in order to set the remote S3 encryption scheme as `sse-s3`, create an app with the config in indexes.conf file under default/local sub-directory as follows:
+```
+[volume:\<remote_volume_name\>]]
+path = <remote_volume_path>
+remote.s3.encryption = sse-s3
+```
+2. Apply the CR with the necessary & supported Smartstore and Index related configs
+3. Install the App created using the [currently supported methods](https://splunk.github.io/splunk-operator/Examples.html#installing-splunk-apps)
```

Comment:
Can 2&3 be part of one step right?. Customers can apply the CR at once for both.

Reply:
Agreed, they can be. By listing the steps separately, I am just trying to clarify that most aspects of the config still remain in the CR while the delta is part of the app.

Reply:
Does it look better now, @sgontla ?

Reply:
Looks good to me :-)